llvm/lib/Target/PowerPC/PPCFrameLowering.cpp

```diff
@@ -2399,6 +2405,43 @@ bool PPCFrameLowering::assignCalleeSavedSpillSlots(
   return AllSpilledToReg;
 }
 
+static void findContinuousLoadStore(const MachineFunction *MF,
+                                    ArrayRef<CalleeSavedInfo> CSI,
+                                    Register &MergeFrom) {
+  const MachineFrameInfo &MFI = MF->getFrameInfo();
+  int64_t Offset = MFI.estimateStackSize(*MF);
```

Comment:
Could we use `PPCFrameLowering::determineFrameLayout()` to determine the stack size on PPC?